scripts/lib/tests/test_dev_env_optimize.py

```
@@ -0,0 +1,154 @@
+    env.update({
+        "PATH": f"{bin_dir}:{env['PATH']}",
+        "PROJECT_ROOT": str(project_root),
+        "PROC_ROOT": str(proc_root),
+        "KILL_LOG": str(kill_log),
+        "COMMAND_LOG": str(command_log),
+        "KILL_COMMAND": str(bin_dir / "record-kill"),
+        "KEEP_RA": "2",
+        "CARGO_ZOMBIE_MIN": "30",
+    })
+
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)